photonpump/connection.py

```diff
@@ -722,7 +740,7 @@ def close(self):
         cmd = convo.ConnectPersistentSubscription(
             subscription, stream, credentials=self.credential
         )
-        future = await self.protocol.enqueue_conversation(cmd)
+        future = await self.protocol.enqueue_conversation(cmd, True)
```

Comment:
Seems like there should be a test covering this code?

Reply:
added a test for this.